src/screens/Exchange/ExchangeConfirm.js

```diff
@@ -345,12 +351,19 @@ class ExchangeConfirmScreen extends React.Component<Props, State> {
 
               </AllowanceWrapper>
             }
-            <ButtonText
+            {!hasSmartWallet && <ButtonText
               buttonText="Legacy Wallet"
               rightIconProps={{ name: 'selector', style: { fontSize: 16 } }}
```

Comment:
Shouldn't that button allow to select where do you want to receive your assets?

Reply:
With the latest changes the smart wallet will be the default receiver (image1). We agreed to have this button only to guide users (edge-case legacy only) to create a smart wallet account. (images 2, 3)